web/src/components/storage/DASDTable.tsx

```diff
@@ -88,6 +92,11 @@ const columns = [
 ];
 
 const Actions = ({ devices, isDisabled }: { devices: DASDDevice[]; isDisabled: boolean }) => {
+  const { mutate: enableDASD } = useEnableDASDMutation();
```

Comment:
I think using a parameter (instead of having a hook for enabling and another for disabling) might be a better idea and would help to reduce the amount of code.

Reply:
Ok, will address it.